src/app/(food)/details.tsx

```diff
@@ -274,7 +287,7 @@ export default function FoodDetail(): JSX.Element {
                     <Text
                         style={[styles.notesText, { color: colors.labelColor }]}
                     >
-                        {t('details.footer')}
+                        {t(isGerman() ? 'details.footerOrg' : 'details.footer')}
```

Comment:
Unrelated to this PR, but please left align the text. Even Apple uses left-align in this case ;)

![IMG_0004](https://github.com/neuland-ingolstadt/neuland.app-native/assets/39560569/bd517753-6c15-474e-9c9a-a4af372c24dd)